view/SSViewer.php

```diff
@@ -181,6 +189,27 @@ public function __call($name, $arguments) {
 		$this->resetLocalScope();
 		return $retval;
 	}
+
+	/**
+	 * @return array
+	 */
+	protected function getItemStack() {
+		return $this->itemStack;
+	}
+
+	/**
+	 * @param array
+	 */
+	protected function setItemStack(array $stack) {
+		$this->itemStack = $stack;
+	}
```

Comment:
The alternative to adding a setter was to use references to modify the property directly:

```php
protected function &getItemStack() {
    return $this->itemStack;
}

// ...

$itemStack =& $this->getItemStack();
$itemStack[$this->getUpIndex()][SSViewer_Scope::ITEM_OVERLAY] = $this->overlay;
```